src/styles/blocks/header.scss

```diff
@@ -38,5 +38,11 @@
     text-transform: uppercase;
     font-size: 12px;
     font-weight: 500;
+
+    transition: color 0.3s;
```

Comment:
Good job! 👍
Maybe, it would be better to add some variables...
Take a look at the requirements to the task:
![image](https://user-images.githubusercontent.com/127268424/235374642-be0ecce9-65e3-4441-bc8d-b9b1187a0249.png)

Reply:
Thanks, done